### PR TITLE
Added Server starter

### PR DESCRIPTION
**Pull request by Trion129**
Please review I have some doubts on correctness

### CODE REVIEW

File: build/dev-server.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ var autoOpenBrowser = !!config.dev.autoOpenBrowser
 // https://github.com/chimurai/http-proxy-middleware
 var proxyTable = config.dev.proxyTable
 
-var app = express()
+var app = require('../server/index.js')
 var compiler = webpack(webpackConfig)
 
 var devMiddleware = require('webpack-dev-middleware')(compiler, {
```
Comment (Oxyrus) [inline build/dev-server.js]: That's not expected to be used.

Comment (Trion129) [reply build/dev-server.js]: So server and client should be completely seperate?

Comment (Trion129) [reply build/dev-server.js]: So server and client should be completely seperate?

Comment (Oxyrus) [reply build/dev-server.js]: That would probably be the best idea, but I haven't had time to check SSR on Vue.

**Comment by Trion129** [comment]
Stale PR, I'll setup server again later, closing for now

**Pull request closed without merging**